Multisig/UI/UI Library/Views/AddressInfoView.swift

```diff
@@ -80,13 +89,26 @@ class AddressInfoView: UINibView {
     @IBAction private func didTouchUp(sender: UIButton, forEvent event: UIEvent) {
         alpha = 1.0
     }
-}
 
+    @objc func displayAddress() {
+        if let _ = label {
+            addressLabel.text = prefixString() + self.address.ellipsized()
+        } else {
+            let prefixString = prefixString()
+            addressLabel.attributedText = (prefixString + self.address.checksummed).highlight(prefix: prefixString.count + 4)
+        }
+    }
+
+    private func prefixString() -> String {
```

Comment:
should this be extracted and reused? seems repeated several times

Reply:
Actually what should be done is refactor AddressInfoView + SafeBarView, as they are almost identical, but will keep this for later